lib/Cargo.toml

```diff
@@ -32,6 +32,8 @@ tracing = "0.1"
 ureq = "2"
 url = "2"
 urlencoding = "2"
+futures = "0.3.21"
```

Comment:
I don't see the dependencies being used anywhere yet. Am I overlooking something?

Reply:
nope, work in progress slipped in.